code/modules/mob/living/emote.dm

```diff
@@ -326,7 +326,7 @@
 /datum/emote/living/surrender
 	key = "surrender"
 	key_third_person = "surrenders"
-	message = "puts their hands on their head and falls to the ground, they surrender%s!"
+	message = "puts their hands on their head and falls to the ground, they surrender!"

```

Comment:
Well now I need to break the pronoun defines so I have mexican standoffs to crusade for.